frontend/test/pytest/test_mid_circuit_measurement.py

```diff
@@ -240,9 +294,10 @@ def circuit(x):
     @pytest.mark.parametrize(
         "meas_obj", [qml.PauliZ(0), qml.Hadamard(0) @ qml.PauliZ(1), [0], [0, 1], "mcm"]
     )
+    @pytest.mark.parametrize("use_kwarg", [True, False])
```

Comment:
I would expect this to be costly. Could we skip most tests for one of them?

Reply:
If we want to make the use of `mcm_method` the default way to request MCM transforms, then I'll update the tests to use that instead of decorating with `dynamic_one_shot`.